gradle-typescript/src/main/java/com/gradlets/gradle/typescript/TypeScriptPlugin.java

```diff
@@ -204,14 +207,24 @@ private static void addModule(
                         project.getObjects().named(LibraryElements.class, TypeScriptAttributes.MODULE));
         moduleVariant.artifact(moduleArtifact);
 
-        NamedDomainObjectContainer<ConfigurationVariant> runtimeVariants = publications.getVariants();
-        ConfigurationVariant resourcesVariant = runtimeVariants.create("package-json");
+        ConfigurationVariant resourcesVariant = publications.getVariants().create("package-json");
         resourcesVariant.getAttributes().attribute(ArtifactAttributes.ARTIFACT_FORMAT, "package-json");
         resourcesVariant
                 .getAttributes()
                 .attribute(
                         LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE,
                         project.getObjects().named(LibraryElements.class, TypeScriptAttributes.PACKAGE_JSON));
         resourcesVariant.artifact(packageJsonArtifact);
+
+        ConfigurationVariantInternal variant = (ConfigurationVariantInternal)
+                configuration.getOutgoing().getVariants().create("source-script-dirs");
+        variant.getAttributes().attribute(ArtifactAttributes.ARTIFACT_FORMAT, TypeScriptAttributes.MODULE);
```

Comment:
I don't think this counts as a module anymore

Reply:
sure, it could be that I forgot to remove something from my testing.